### PR TITLE
Create leaflet-coloricon.md

### PR DESCRIPTION
**Pull request by shevekk**

### CODE REVIEW

File: docs/_plugins/markers-renderers/leaflet-coloricon.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Leaflet.ColorIcon
+category: markers-renderers
+repo: https://github.com/shevekk/Leaflet.ColorIcon
+author: Maxence Martin (shevek)
+author-url: https://github.com/shevekk
+demo: http://dataexplorer.hd.free.fr/Leaflet.ColorIcon/examples/basic/
+compatible-v0:
```
Comment (Falke-Design) [inline docs/_plugins/markers-renderers/leaflet-coloricon.md:8]: Is your plugin created for V0? I think not but please confirm with changing to `compatible-v0: false`

File: docs/_plugins/markers-renderers/leaflet-coloricon.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+compatible-v1: true
+---
+
+Create colored icons, allows to generate a css filter on the icon
```
Comment (Falke-Design) [inline docs/_plugins/markers-renderers/leaflet-coloricon.md:12]: Maybe you want to change the text to something like that:

> Overwrite color of the icon with CSS filters and make it possible to add same icon-image in different colors.

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Nice plugin, I need exact this feature and thought I must write it by my self. I will give it a try 👍

Please resolve my comments below.

File: docs/_plugins/markers-renderers/leaflet-coloricon.md (hunk 2 above)
Comment (shevekk) [reply docs/_plugins/markers-renderers/leaflet-coloricon.md]: Thank you for your suggestion, I have changed the description

**Review by Falke-Design — APPROVED** [review]

**Review by johnd0e — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**